lib/wicked/TestInfo.pm

```diff
@@ -0,0 +1,20 @@
+# Copyright © 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+package wicked::TestInfo;
```

Comment:
Maybe we could call it `TestContext` or similar?
And call it `$ctx` in the run() method.
Wdyt?

Reply:
yeah we can call it `TestContext` or anything else. My idea why I call it `TestInfo` that LTP sibling with same purpose and actually prototype of this one  ( copy/paste prototype :D ). Has similar name https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/LTP/TestInfo.pm

Reply:
Ok I see. 
I think in ltp case it is realy the info, as it contains the testlist which need to be run.

I also do not have hard feelings here, only `$args` would be nice to replaced by something more similar to the object, what ever name it will have in the end.

Reply:
done